definitions.md

```diff
@@ -101,6 +105,12 @@ The Generation Attack Threshold is equal to the Polling Time + some Grace Period
 
 #### GHOST
 
+GHOST is an acronym for `Greedy Heaviest Observable SubTree`, a class of blockchain structures in which multiple blocks can validly be included in the chain at any given height or round. GHOSTy protocols produce blockDAGs rather than blockchains and use a weighting function for fork selection, rather than simply picking the longest chain.
+
+#### Height
+
+`Height` refers to the number of `TipSets` that have passed between this `TipSet` and the genesis block (which starts at block height 0). If a `TipSet` contains multiple blocks, each block in the TipSet will have the same `height`.
```

Comment:
please say how this interacts with null blocks here

Reply:
FWIW the go-filecoin code, and everyones intuitions, say that "height" is the number of rounds, not the number of tipsets. We'll be swimming upstream to redefine that here. At present, the code doesn't have a use for a count of the non-null tips.

Reply:
to be handled in a different PR per https://github.com/filecoin-project/specs/pull/351